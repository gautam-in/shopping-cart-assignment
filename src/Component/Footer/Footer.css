.App footer {
  height: 20px;
  position: fixed;
  left: 0;
  bottom: 0;
  width: 100%;
  background-color: #eaeaea;
  color: black;
  text-align: center;
}
