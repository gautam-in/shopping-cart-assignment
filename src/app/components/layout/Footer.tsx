import React from 'react';

const Footer = () => {
    return (
        <div className='bg-gray-200 fixed bottom-0 h-8 w-full text-center p-1 text-gray-600'  >
            Copyright @2011-2023 Sabka Bazaar Grocery Supplies Pvt. Ltd.
        </div>
    );
}

export default Footer;