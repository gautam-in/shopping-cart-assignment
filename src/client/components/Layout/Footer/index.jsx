import React from 'react';
import withStyles from 'isomorphic-style-loader/withStyles';

import styles from './Footer.scss';

const Footer = () => (
  <footer>
    <p>Copyright &copy; 2011-2018 Sabka Bazaar Grocery Supplies Pvt Ltd</p>
  </footer>
);

export default withStyles(styles)(Footer);
