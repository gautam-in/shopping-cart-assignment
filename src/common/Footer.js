import React from 'react'

function Footer() {
  return (
    <div className="footer pg-lr-sp">Copyright 2011-2018 Sabka Bazaar Grocery Supplies Pvt. Ltd.</div>
  )
}

export default Footer