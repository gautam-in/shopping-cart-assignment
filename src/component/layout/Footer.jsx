import React from "react";

function Footer() {
  return (
    <footer className="app-footer">
      <b>Copyright Â© 2011-2021 Sabka Bazaar Grocery Supplies PVT LTD</b>
    </footer>
  );
}

export default Footer;
