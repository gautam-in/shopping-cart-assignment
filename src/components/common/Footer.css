.footer {
    width: 100%;
    background-color: #e7e7ec;
    padding: 0.01%;
}

.footer>p {
    margin-left: 16%;
}