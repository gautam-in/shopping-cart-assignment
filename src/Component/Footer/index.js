import React from "react";
import "./Footer.css"
export default function index() {
  return (
    <footer>
      <p>Copyright @2011-2018 Sabka Bazaar Grocery Supplies Pvt Ltd</p>
    </footer>
  );
}
