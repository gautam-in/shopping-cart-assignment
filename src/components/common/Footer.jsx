import React from 'react'
import './Footer.css'
const Footer = () => {
    return (
        <footer className="footer">
            <p>
                Copyright (c) 2020-2021 Sabka Bazaar Grocery Supplies Pvt Ltd
            </p>
        </footer>)
}

export default Footer;