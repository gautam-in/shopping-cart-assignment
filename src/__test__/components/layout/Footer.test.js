import { shallow,configure } from "enzyme";
import Adapter from 'enzyme-adapter-react-16';
import  Footer from '../../../components/layout/Footer';
  
configure({ adapter: new Adapter() })


describe('Footer Snapshots', () => {
    it('should render our Snapshots correctly', () => {
        const wrapper = shallow(<Footer />);
        expect(wrapper).toMatchSnapshot();
    });

    it('render text', () => {
        const wrapper = shallow(<Footer />);
        expect(wrapper.text()).toEqual(' Copyright Â© 2011-2018 Sabka Bazaar Grocery Supplies Pvt Ltd.');
      });
});