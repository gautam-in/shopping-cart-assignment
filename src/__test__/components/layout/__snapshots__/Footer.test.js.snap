// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator Snapshots should render our Snapshots correctly 1`] = `ShallowWrapper {}`;

exports[`Footer Snapshots should render our Snapshots correctly 1`] = `ShallowWrapper {}`;
